pkg/agent/openflow/pipeline.go

```diff
@@ -2426,6 +2433,7 @@ func (c *client) endpointDNATFlow(endpointIP net.IP, endpointPort uint16, protoc
 			&binding.PortRange{StartPort: endpointPort, EndPort: endpointPort},
 		).
 		LoadToCtMark(ServiceCTMark).
+		MoveToCtMark(PktSourceField.GetNXFieldName(), PktSourceField.GetRange(), &binding.Range{0, 3}).
```

Comment:
Should the flows in `ConntrackCommit` be updated to be consistent? Otherwise it would happen again some packets have some ct marks while other don't.
The following 3 flows in `ConntrackCommit` can be merged to one loading reg0 to CTMark.
```
table=ConntrackCommit, priority=200,ct_state=+new+trk,ip,reg0=0x1/0xf actions=ct(commit,table=HairpinSNAT,zone=65520,exec(load:0x1->NXM_NX_CT_MARK[1]))
table=ConntrackCommit, priority=200,ct_state=+new+trk,ip,reg0=0x5/0xf actions=ct(commit,table=HairpinSNAT,zone=65520,exec(load:0x1->NXM_NX_CT_MARK[3]))
table=ConntrackCommit, priority=190,ct_state=+new+trk,ip actions=ct(commit,table=HairpinSNAT,zone=65520)
```

Reply:
Reasonable.